.github/workflows/CI-rpc-tests.yml

```diff
@@ -16,59 +16,98 @@ jobs:
 
       - name: Build Docker image
         id: build_image
-        run: docker build -t rpc-tests:latest -f src/rpc-tests/Dockerfile .
+        run: docker build --no-cache -t rpc-tests:latest -f src/rpc-tests/Dockerfile .
         working-directory: .
 
       - name: Run RPC tests
         id: run_tests
         run: |
           mkdir -p src/rpc-tests/reports
-          docker run --rm -e SEED_PHRASE=${{ secrets.QA_SEED_PHRASE }} -v ${{ github.workspace }}/src/rpc-tests/reports:/usr/src/app/reports rpc-tests:latest 2>&1 | tee src/rpc-tests/reports/test_output.txt
+          echo "Starting Docker container for RPC tests..."
+          
+          docker run --rm \
+            -e SEED_PHRASE_1=${{ secrets.QA_SEED_PHRASE_1 }} \
+            -e SEED_PHRASE_2=${{ secrets.QA_SEED_PHRASE_2 }} \
+            -e SEED_PHRASE_3=${{ secrets.QA_SEED_PHRASE_3 }} \
+            -e SEED_PHRASE_4=${{ secrets.QA_SEED_PHRASE_4 }} \
+            -e SEED_PHRASE_5=${{ secrets.QA_SEED_PHRASE_5 }} \
+            -e SEED_PHRASE_6=${{ secrets.QA_SEED_PHRASE_6 }} \
+            -v ${{ github.workspace }}/src/rpc-tests/reports:/usr/src/app/reports \
+            rpc-tests:latest 2>&1 | tee src/rpc-tests/reports/test_output.txt
```

Comment:
Use quotes, as the seed phrase env variables have spaces:
```
        env:
          QA_SEED_PHRASE_1: "${{ secrets.QA_SEED_PHRASE_1 }}"
          QA_SEED_PHRASE_2: "${{ secrets.QA_SEED_PHRASE_2 }}"
          QA_SEED_PHRASE_3: "${{ secrets.QA_SEED_PHRASE_3 }}"
          QA_SEED_PHRASE_4: "${{ secrets.QA_SEED_PHRASE_4 }}"
          QA_SEED_PHRASE_5: "${{ secrets.QA_SEED_PHRASE_5 }}"
          QA_SEED_PHRASE_6: "${{ secrets.QA_SEED_PHRASE_6 }}"
        run: |
          mkdir -p src/rpc-tests/reports
          echo "Starting Docker container for RPC tests..."
          
          docker run --rm \
            -e SEED_PHRASE_1="$QA_SEED_PHRASE_1" \
            -e SEED_PHRASE_2="$QA_SEED_PHRASE_2" \
            -e SEED_PHRASE_3="$QA_SEED_PHRASE_3" \
            -e SEED_PHRASE_4="$QA_SEED_PHRASE_4" \
            -e SEED_PHRASE_5="$QA_SEED_PHRASE_5" \
            -e SEED_PHRASE_6="$QA_SEED_PHRASE_6" \
            -v ${{ github.workspace }}/src/rpc-tests/reports:/usr/src/app/reports \
            rpc-tests:latest 2>&1 | tee src/rpc-tests/reports/test_output.txt
```

Reply:
Not sure about this one - is it failing somewhere that this change is needed?